lms/djangoapps/grades/tests/test_transformer.py

```diff
@@ -330,6 +332,37 @@ def test_course_version_collected_in_split(self):
         block_structure = get_course_blocks(self.student, blocks[u'course'].location, self.transformers)
         self.assertIsNotNone(block_structure.get_xblock_field(blocks[u'course'].location, u'course_version'))
 
+    def test_grading_policy_collected(self):
+        blocks = self.build_course_with_problems()
+        block_structure = get_course_blocks(self.student, blocks[u'course'].location, self.transformers)
+
+        # the calculated hash of the grading policy of the test course, before and after updates
+        expected_grading_policy_hash = u'ChVp0lHGQGCevD0t4njna/C44zQ='
+        expected_updated_grading_policy_hash = u'TsbX04qWOy1WRnC0NHy+94upPd4='
+
+        self.assert_collected_transformer_block_fields(
+            block_structure,
+            blocks[u'course'].location,
+            self.TRANSFORMER_CLASS_TO_TEST,
+            grading_policy_hash=expected_grading_policy_hash,
+        )
```

Comment:
Additionally, can you verify that the recomputed hash equates when the course policy isn't changed?